packages/fx-core/tests/plugins/resource/bot/unit/languageStrategy.test.ts

```diff
@@ -31,10 +31,15 @@ describe("Language Strategy", () => {
       sinon.stub(LanguageStrategy, "getTemplateProjectZipUrl").resolves("");
 
       // Act
-      const zip = await LanguageStrategy.getTemplateProjectZip(programmingLanguage, group_name);
+      try {
+        await LanguageStrategy.getTemplateProjectZip(programmingLanguage, group_name);
+      } catch (e) {
+        chai.assert.isTrue(e instanceof PluginError);
```

Comment:
In my understanding, this would fail if the zip has been downloaded.

Reply:
Let me update it with mock-fs.